### ISSUE TITLE
Allow on the fly changes to config.toml using workflows

### ISSUE THREAD

**Issue by l1meju1ce** [porla/porla#180]
Give workflows the ability to change settings in the config.toml that are loaded without needing to restart porla.

This would allow changes in download/upload_rate_limit depending on the time of day for example. Also the listen_interface if the port forwarding from the VPN provider changes.

**Comment by vktr** [comment]
I'm thinking the actual config file should be read only, but we should be able to change the values in-memory while running. Something like,

```lua
function(ctx)
  local settings = ctx.session.settings()
  settings.download_rate_limit = 1024
  settings.upload_rate_limit = 1024
end
```

At least, this is how I'm imagining things until we move all settings to the database - when we've moved there, it will be easier to do things like persist changes from Lua.

**Comment by l1meju1ce** [comment]
Yeah that makes sense, basically being able to change settings and load them into porla on the fly without having to restart the client.

**Comment by vktr** [comment]
I think this can be closed now that #233 has been merged. It supports getting and setting session settings.
